### DISCUSSION TITLE
Next frame issue

### DISCUSSION THREAD

**Discussion by MunzyTdub** [keiwando/toonsquid#137] [Q&A]
Hi, can someone help me understand why when i want to auto add a new frame it adds about 5 seconds instead of just one frame?

**Comment by Cat-Butler** [comment]
Actions > Settings > General > choose Frames or Drawings mode > Default drawing length (in frames).
Change it to 1 frame, or whatever you prefer.

I believe the default app setting for Drawings mode (as opposed to Frames, or is it the other way around?) is 5 seconds.

This is set up for “draw once (or import) then transform” (use movement, scaling, rotation - great for motion graphics, flying objects, mechanical motion and physics stuff, hierarchical character motion), as opposed to frame-by-frame drawing.

**Reply by MunzyTdub** [reply]
Many thanks :D

**Comment by keiwando** [comment]
Just to add to @Cat-Butler's answer, here's the relevant section of the handbook that explains the different default project settings that are chosen based on your choice of the "Default Workflow" when you create a new project:

https://toonsquid.com/handbook/projects/create_project/#workflow

And this is the setting with which you can change the default duration of drawings:
https://toonsquid.com/handbook/settings/general/#default-drawing-length

**Reply by MunzyTdub** [reply]
Thank you. Love what you've made btw. Toonsquids really great!
